rpc/wrpc/beacon/src/connection.rs

```diff
@@ -0,0 +1,209 @@
+use crate::imports::*;
+
+const BIAS_SCALE: u64 = 1_000_000;
+
+#[derive(Debug)]
+pub struct Connection {
+    pub node: Arc<Node>,
+    bias: u64,
+    output: RwLock<String>,
+    sender: Sender<Params>,
+    client: KaspaRpcClient,
+    shutdown_ctl: DuplexChannel<()>,
+    connected: Arc<AtomicBool>,
+    is_synced: Arc<AtomicBool>,
+    online: Arc<AtomicBool>,
+    clients: Arc<AtomicU64>,
```

Comment:
these arcs are unnecessary and never used

Reply:
no, they are... all good.

Reply:
Do you mean you will clone them in future? cuz rn there's no reason to wrap Atomics into Arc